database-commons/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java

```diff
@@ -110,9 +110,14 @@ public void close(TaskAttemptContext context) throws IOException {
         }
 
         @Override
-        public void write(K key, V value) throws IOException {
-          super.write(key, value);
+        public void write(K key, V value) {
           emptyData = false;
+          try {
+            key.write(getStatement());
```

Comment:
can you add a comment about why super.write() is not used? It seems like it's because the exception gets lost?

Reply:
this is used instead of super method to use proper logging.